src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -619,6 +626,61 @@ class NetworkOPsImp final
     std::vector <TransactionStatus> mTransactions;
 
     StateAccounting accounting_ {};
+
+private:
+    struct Stats
+    {
+        template <class Handler>
+        Stats (Handler const& handler, beast::insight::Collector::ptr const& collector)
+            : hook (collector->make_hook (handler))
+            , disconnected_duration (collector->make_gauge("State_Accounting","Disconnected_duration"))
+            , connected_duration (collector->make_gauge("State_Accounting","Connected_duration"))
+            , syncing_duration(collector->make_gauge("State_Accounting", "Syncing_duration"))
+            , tracking_duration (collector->make_gauge("State_Accounting", "Tracking_duration"))
+            , full_duration (collector-> make_gauge("State_Accounting", "Full_duration"))
+            , disconnected_trasitions (collector->make_gauge("State_Accounting","Disconnected_trasitions"))
+            , connected_trasitions (collector->make_gauge("State_Accounting","Connected_trasitions"))
```

Comment:
`transitions` is misspelled in both the member and gauge names.

Reply:
Oops, thanks.